packages/flutter_tools/test/commands.shard/hermetic/doctor_test.dart

```diff
@@ -816,6 +818,166 @@ void main() {
   }, overrides: <Type, Generator>{
     AndroidWorkflow: () => FakeAndroidWorkflow(appliesToHostPlatform: false),
   });
+
+  group('Doctor events with unified_analytics', () {
+    late FakeAnalytics fakeAnalytics;
+    final FakeFlutterVersion fakeFlutterVersion = FakeFlutterVersion();
+    final SystemClock fakeSystemClock = SystemClock.fixed(DateTime(1995, 3, 3));
+
+    setUp(() {
+      fakeAnalytics = getInitializedFakeAnalyticsInstance(
+        fakeFlutterVersion: fakeFlutterVersion,
+        fs: fs,
+      );
+    });
```

Comment:
All of the tests in this group are the same tests that existed for usage events that were sent to GA3 except refactored to work with the new analytics package... link below to where these tests are in this file

https://github.com/eliasyishak/flutter/blob/sending-doctor-events-ga4/packages/flutter_tools/test/commands.shard/hermetic/doctor_test.dart#L187-L310